docs/guide/website/docs/doc1.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
this is pretty much docusaurus get started, I'm pretty sure we don't need this ><